lib/rules/spaced-comment.js

```diff
@@ -111,9 +113,8 @@ function createAlwaysStylePattern(markers, exceptions) {
             pattern += exceptions.map(escapeAndRepeat).join("|");
             pattern += ")";
         }
-        pattern += "$)"; // the sequence continues until the end.
+        pattern += (ending + ")"); // the sequence continues until the end.
```

Comment:
I'll give it a shot, thought it may have side effects with multi-line line comments. let me check...